components/aws/sagemaker/batch_transform/src/batch_transform.py

```diff
@@ -56,10 +51,9 @@ def main(argv=None):
   batch_job_name = _utils.create_transform_job(client, vars(args))
   logging.info('Batch Job request submitted. Waiting for completion...')
   _utils.wait_for_transform_job(client, batch_job_name)
-  _utils.print_tranformation_job_result(args.output_location)
 
-  Path(args.output_location_file).parent.mkdir(parents=True, exist_ok=True)
-  Path(args.output_location_file).write_text(unicode(args.output_location))
+  with open('/tmp/output_location.txt', 'w') as f:
```

Comment:
The original lines might be better for future-proof components. We've already got some bug reports about issues stemming from hard-coded paths causing problems for certain on-prem scenarios such as non-Docker OCI clusters.

Reply:
Got it, went back to the original way